src/components/atm.input/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { useState } from 'react';
+import { IconBaseProps } from 'react-icons';
+
+interface InputProps extends React.InputHTMLAttributes<HTMLInputElement> {
+  labelName?: string;
+  icon?: React.ComponentType<IconBaseProps>;
+}
+
+export function Input({labelName, icon: Icon, ...rest}: InputProps) {
```

Comment:
acho que o icon pode deixar com minúsculo mesmo: coisas que iniciam com letra maiúscula são nomes de classes ou de interfaces
```suggestion
export function Input({labelName, icon, ...rest}: InputProps) {
```

Reply:
Não consegui fazer com o icon minúsculo, pq preciso dele maiusculo (Icon) para poder renderizar este icone

Reply:
E ao mesmo tempo, não queria começar uma prop/interface com primeira letra maiuscula, por isso fiz a transformação. Existe alguma maneira melhor?